docs/middleware/logger.md

```
@@ -88,6 +88,27 @@ app.Use(logger.New(logger.Config{
 app.Use(logger.New(logger.Config{
     DisableColors: true,
 }))
+
+// Use Custom Logger with Fiber Logger Interface
+func LoggerToWriter(customLogger fiberlog.AllLogger, level fiberlog.Level) io.Writer {
+	return &struct {
```

Markdownlint (0.35.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)